### PR TITLE
US-4361: HTML for Streaming Landing Page

### PR DESCRIPTION
**Pull request by tcmacdonald**
This PR delivers [US-4361](https://rally1.rallydev.com/#/41662702253d/detail/userstory/58571943886) by implementing static HTML/CSS for the updated streaming landing page. 

Since this work originated as a sub-branch of [`US4291-bootstrap-for-angular2`](https://github.com/crdschurch/crds-angular/tree/US4291-bootstrap-for-angular2) this PR is attempting to merge these updates back into that branch vs `development` but let me know if you'd prefer this be handled in another fashion. 

**NOTE:** 

Some of the elements here are still being reviewing / receiving feedback so some items may not match the concepts and/or prototype exactly. We're operating under the assumption that the following things will need to happen after this code has been merged... 

- [ ] [US-4411](https://rally1.rallydev.com/#/41662702253d/detail/userstory/58960061497) – Implement Typekit account for Crossroads and replace the existing embed
- [ ] [US-4412](https://rally1.rallydev.com/#/41662702253d/detail/userstory/58961988131) – Implement some simple CSS animation (pending approval of wow.js) /cc @bzmillerboy 
- [ ] Determine global relevance of any variables / mixins / elements used herein &amp; move them to their appropriate place where appropriate. /cc @bzmillerboy 

crds-anywhere-landing-01

### CODE REVIEW

File: crossroads.net/app/index.html [hunk 1]
```diff
@@ -85,9 +85,15 @@
     <script>
         window.prerenderReady = false;
     </script>
+
+    <!-- TODO Replace w/ production TypeKit account -->
```
Comment (bzmillerboy) [inline crossroads.net/app/index.html]: @tcmacdonald Did you create a story in Rally for this. If not, please do so we don't loose track of it.

File: crossroads.net/styles/pages/streaming.scss [hunk 2]
```diff
@@ -0,0 +1,380 @@
+// TODO Replace w/ CRDS endpoint once we have images in production S3 bucket.
+$streaming-imgix-domain: '//ample.imgix.net';
+
+// NOTE These vars aren't assumed to be global yet, hence the `streaming` prefix.
```
Comment (bzmillerboy) [inline crossroads.net/styles/pages/streaming.scss]: I'd rather these be global if we're going to make the change here.

File: crossroads.net/styles/pages/streaming.scss [hunk 3]
```diff
@@ -0,0 +1,380 @@
+// TODO Replace w/ CRDS endpoint once we have images in production S3 bucket.
```
Comment (bzmillerboy) [inline crossroads.net/styles/pages/streaming.scss]: @tcmacdonald Do you need access to our S3 bucket? For now you can publish the images through our CMS (do you have access to that?)

File: crossroads.net/app/index.html (hunk 1 above)
Comment (tcmacdonald) [inline crossroads.net/app/index.html]: Indeed– https://rally1.rallydev.com/#/41662702253d/detail/userstory/58960061497

File: crossroads.net/styles/pages/streaming.scss (hunk 3 above)
Comment (tcmacdonald) [inline crossroads.net/styles/pages/streaming.scss]: Captured this via [US-4419](https://rally1.rallydev.com/#/41662702253d/detail/userstory/58984628724?fdp=true)

**Pull request merged**